packages/boot/test/bootstrapTests/supports.js

```diff
@@ -188,9 +188,7 @@ export const getNodeTestVaultsConfig = async (
   bundleDir = 'bundles',
   specifier = '@agoric/vm-config/decentral-itest-vaults-config.json',
 ) => {
-  const fullPath = await importMetaResolve(specifier, import.meta.url).then(
-    u => new URL(u).pathname,
-  );
+  const fullPath = pkgAbsPath(specifier);
```

Comment:
presuming the `specifier` arg is always a design-time constant, sync access is OK here. I'd be OK with copying `pkgAbsPath` into this module.

Reply:
My preference is to inline `url.fileURLToPath(importMetaResolve(specifier, import.meta.url))` here with the expectation that this will eventually become `url.fileURLToPath(import.meta.resolve(specifier))` when we drop support for Node.js versions that lack `import.meta.resolve`.

I also encourage a general move toward framing Agoric APIs in terms of URLs instead of paths (as in Compartment Mapper) since they are more portable and obviate any impulse toward making this utility function.